### PR TITLE
Proposed Typography/Layout

### PR DESCRIPTION
**Pull request by IshaDakota**
Some minor layout and typography.

### CODE REVIEW

**Comment by amitaibu** [comment]
Render are going to do some work on the site, so I think design (vs conceptual) efforts can be on hold.

**Comment by IshaDakota** [comment]
I'm done - just removed the WIP tags to indicate as such (and re-roll to keep current).

What's the timeline? been days no movement.

**Comment by amitaibu** [comment]
No eta, we should get an offer beginning of next week.

**Comment by amitaibu** [comment]
> I'm done - just removed the WIP tags to indicate as such

Can you attach screenshot(s) please

**Comment by IshaDakota** [comment]
Most of what is in this PR is in other as well:

typography:
project_portfolio

blog/case study layout:
i_shadowed_our_cto_for_a_day__and_this_is_what_i_learned

left-aligned story title:
clean_entry

Using rem makes lists consistent:
open_data_and_our_commit_ment_

**Comment by amitaibu** [comment]
all seems good, just the `very padded` might look a bit too padded in mobile?

**Comment by IshaDakota** [comment]
>  the very padded might look a bit too padded in mobile?

I think you're right - really about line length, the padding of the segment and the extra padding round the text makes it too short on mobile.

**Comment by amitaibu** [comment]
@IshaDakota lets remove the `very padded`, and get this in?

**Comment by IshaDakota** [comment]
> lets remove the very padded

done. I'd encourage you to take a look at #560 , #561 , and #565 as well - I did them together but broke the up into chunks. You can see all together in a branch called [experimental](https://github.com/Gizra/gizra.com/tree/experimental).

File: src/_includes/project.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="ui segment projects">
+<div class="ui very padded segment projects">
```
Comment (amitaibu) [inline src/_includes/project.html]: I'll remove this one, and merge

**Comment by amitaibu** [comment]
Oh, seems a project page now has an issue with the label

http://localhost:3000/case-studies/openscholar/

one_database_to_rule_them_all_

**Comment by IshaDakota** [comment]
> Oh, seems a project page now has an issue with the label

fixed

one_database_to_rule_them_all

**Comment by amitaibu** [comment]
Merged, thanks.

**Pull request merged**